recipe/0309-annotation-collection/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Using Annotation Collections
```

Comment:
The title seems more like a tutorial than a recipe. If I were looking for this, I would be searching for terms from the use case